ehealthid-rp/src/test/java/com/oviva/ehealthid/relyingparty/svc/TokenIssuerImplTest.java

```diff
@@ -210,6 +212,153 @@ void redeem_idToken() throws JOSEException, ParseException {
     assertIdTokenClaims(jws, nonce, issuer, clientId);
   }
 
+  @Test
+  void issueIdToken_gemSpecA23035() throws JOSEException, ParseException {
+    var issuer = URI.create("https://idp.example.com");
+
+    var k = genKey();
+    var keyStore = mock(KeyStore.class);
+    when(keyStore.signingKey()).thenReturn(k);
+    var codeRepo = mock(CodeRepo.class);
+
+    var sut = new TokenIssuerImpl(issuer, keyStore, codeRepo);
+
+    var id = UUID.randomUUID().toString();
+
+    var nonce = UUID.randomUUID().toString();
+    var redirectUri = URI.create("https://myapp.example.com/callback");
+    var clientId = "myapp";
+    var federatedSub = "S12349345";
+    var federatedIss = "https://test-issuer.example.com";
+
+    var federatedIdToken =
```

Comment:
For reasons, I do not understand, `spotless:apply` formats this pretty differently than the other instances in this file

Reply:
interesting, don't know either why :/